crates/basilica-sdk-python/python/basilica/training/__init__.py

```
@@ -0,0 +1,606 @@
+        """Base model name."""
+        return self._base_model
+
+    @property
+    def step_count(self) -> int:
+        """Current training step count."""
+        return self._step_count
+
+    def forward_backward(
+        self,
+        data: List[Datum],
+        loss_fn: str = "cross_entropy",
```

Ruff (0.14.8)

Unused method argument: `loss_fn`

(ARG002)